src/universal-app/kitchen-sink/kitchen-sink.ts

```diff
@@ -145,7 +145,6 @@ export class KitchenSink {
   ],
   declarations: [KitchenSink, TestEntryComponent],
   exports: [KitchenSink, TestEntryComponent],
-  entryComponents: [TestEntryComponent],
```

Comment:
Is this one still necessary since we run it w/ view-engine on CI?

Reply:
Looks like it is. I'll put it back.